NEWS.md

```diff
@@ -68,6 +68,8 @@ TODO: sort out what happens to `glance.aov()`
 
 ## New tidiers, features and bugfixes
 
+- Make fitted values returned by `augment.polr()` match `predict.polr` output. (#619)
```

Comment:
Can we change this to "return factor rather than numeric class predictions in `.fitted` in `augment.polr()`" or something along those lines?